### DISCUSSION TITLE
Colores en nav bar

### DISCUSSION THREAD

**Discussion by gitbraga** [Telefonica/mistica-design#620] [Proposals]
Team: Vivo

---
Desde Vivo, nos gustaria evolucionar la nav bar para tener opción de elegir el color en los flujos o en el header de la tab Inicio.

Empezamos un estudio pero creo que hace falta una definicíon en conjunto con vosotros de cual seria el mejor comportamiento para el caso de utilizar la large nav bar en blanco en flujos que es necesario hacer scroll.

**Comment by yceballost** [comment]
Creamos ticket para tener esto de una vez por todas 😂

**Comment by aweell** [comment]
Hola @gitbraga!

Podrías compartir el estudio que se ha hecho hasta la fecha para poder echarle un vistazo?

**Comment by gitbraga** [comment]
@aweell aqui está https://www.figma.com/file/Ak5swB1EI4QjcGLIStfIeS/Facelift---Quick-Wins?node-id=4630%3A298910&t=DSfPvyFSqSjw25Ij-4

**Comment by yceballost** [comment]
We are upgrading definition for Vivo Hub. This discussion is now deprecated.

We are going to include thre types of variants in navbars: Brand, white & alternative
